src/module.c

```diff
@@ -1227,7 +1284,7 @@ Array reply: a list of the top suggestions matching the prefix
 int SuggestGetCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
   RedisModule_AutoMemory(ctx); /* Use automatic memory management. */
 
-  if (argc < 3 || argc > 8) return RedisModule_WrongArity(ctx);
+  if (argc < 3 || argc > 10) return RedisModule_WrongArity(ctx);
```

Comment:
1-FT.SUGGET 2-key 3-prefix [4-FUZZY] [5-MAX 6-num] [7-WITHSCORES] [8-TRIM] [9-OPTIMIZE] [10-WITHPAYLOADS], I think "MAX num" should be counted by two.